lib/rules/no-shadow-route-definition.js

```diff
@@ -182,25 +186,30 @@ function getPropertyByKeyName(objectExpression, keyName) {
 }
 
 function trimRootLevelNestedRoutes(routesPath) {
-  return routesPath.replace(ROOT_PATH_TRIM_REGEX, '/');
+  return routesPath.replace(ROOT_PATH_TRIM_REGEX, URL_CHUNK_SEPARATOR);
 }
 
 function convertPathToGenericForMatching(routePathInfos) {
   return routePathInfos
     .map((routePathInfo) => {
-      const mappedValue = routePathInfo.stringValue.trim();
-      if (routePathInfo.type === SOURCE_PATH_VALUE_TYPE.STATIC) {
-        if (mappedValue.startsWith('/:')) {
-          return '/:generic-param-for-matching';
-        }
-        if (mappedValue.startsWith('/*')) {
-          return '/*generic-wildcard-for-matching';
-        }
-      }
-      if (routePathInfo.type === SOURCE_PATH_VALUE_TYPE.DYNAMIC) {
-        return `/variable:${mappedValue}`;
-      }
-      return mappedValue;
+      const trimmedValue = routePathInfo.stringValue.trim();
+      const convertedPathForMatchingChunks = trimmedValue
+        .split(URL_CHUNK_SEPARATOR)
```

Comment:
In previous PR for "someVariable" etc. gracefulness, we accidentally removed normalising full paths, this PR brings it back. 🙈